packages/core/README.md

```diff
@@ -0,0 +1,12 @@
+# `@typescript-eslint/core`
+
+> Tooling which enables you to use TypeScript with ESLint
+
+[![NPM Version](https://img.shields.io/npm/v/@typescript-eslint/core.svg?style=flat-square)](https://www.npmjs.com/package/@typescript-eslint/core)
+[![NPM Downloads](https://img.shields.io/npm/dm/@typescript-eslint/core.svg?style=flat-square)](https://www.npmjs.com/package/@typescript-eslint/core)
+
+👉 See **https://typescript-eslint.io/packages/core** for documentation on this package.
```

Comment:
Missing?

Reply:
Coming with the next PR.
The package is private for now so nobody shall see the broken links.